packages/headlamp/v0.24.0+1/package.yaml

```diff
@@ -0,0 +1,27 @@
+# yaml-language-server: $schema=https://glasskube.dev/schemas/v1/package-manifest.json
+
+name: headlamp
+scope: Cluster
```

Comment:
As headlamp has no CRD it could be namespaced, as some hosted cases could similar to quickwit could be possible?

Reply:
nevermind it has a cluster role binding so it needs to stay cluster: https://github.com/headlamp-k8s/headlamp/blob/main/charts/headlamp/templates/clusterrolebinding.yaml